recipe/meta.yaml

```diff
@@ -42,6 +46,10 @@ requirements:
         - libnetcdf
         - kealib
         - krb5
+        - libtiff
+        - libpng
+        - zlib  # [not win]
+        - jpeg  # [not win]
```

Comment:
I am not sure but I guess that the default channel has `zlib` and `jpeg` for Windows, no?